nvflare/private/fed/server/fed_server.py

```diff
@@ -636,7 +636,7 @@ def authentication_check(self, request: Message, state_check):
         if client_ssid != self.server_state.ssid:
             # return make_reply(ReturnCode.AUTHENTICATION_ERROR, "Request from invalid client SSID",
             #                   fobs.dumps(None))
-            error = "Request from invalid client SSID"
+            error = "Request from unknown client SSID"
```

Comment:
Should we make it more detailed? 
"Request has a different SSID than the server side's SSID"

Reply:
Not for now. This change is simply to avoid the word "invalid client" because of the API status determination issue.

Reply:
got it, we might need to enhance this message later.